tests/test_api_build.py

```diff
@@ -796,8 +763,8 @@ def test_about_license_file_and_prelink_message(testing_workdir, testing_config,
 # Regardless of the reason for skipping, we should definitely find a better way for tests to look for the packages
 # Rather than assuming they will be at $ROOT/pkgs since that can change and we don't care where they are in terms of the
 # tests.
-@pytest.mark.xfail(parse_version(conda.__version__) < parse_version("4.3.14"),
-                   reason="new noarch supported starting with conda 4.3.14")
+@pytest.mark.xfail(parse_version(conda.__version__) < parse_version("4.11"),
+                   reason="Older version of not supported conda 4.11")
```

Comment:
We can assume conda is always >= 4.13 per the recipe https://github.com/conda/conda-build/blob/main/recipe/meta.yaml#L29

Reply:
Currently the `conda_interface.py` has no version supporting 4.13 yet.. so did not change that.. these tests will fail if that is not added..

Reply:
`parse_version(conda.__version__) < parse_version("4.11")` will always be False if conda-build is installed correctly.